.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
I don't think you wanted this file to be added.
Could you please remove it and add .`vscode` to .gitgnore?

Reply:
Oops. That's embarrassing :)